x/emissions/module/rewards/rewards_internal.go

```diff
@@ -293,9 +291,21 @@ func GetStakeWeightedLossMatrix(
 			if err != nil {
 				return nil, nil, err
 			}
-			if distance.Gt(maxDistance) {
-				maxDistance = distance
+
+			absoluteDifference := distance
+			if distance.IsNegative() {
+				absoluteDifference, err = distance.Abs()
+				if err != nil {
+					return nil, nil, err
+				}
+			}
```

Comment:
```suggestion
			absoluteDifference, err = distance.Abs()
			if err != nil {
				return nil, nil, err
			}
```

This could be simplified

Reply:
oh yeah, nice catch!